src/apps/common/submodules/portWizard/portWizard.js

```diff
@@ -368,8 +368,9 @@ define(function(require) {
 				}),
 				title: i18n.title,
 				cancel: 'portWizardClose',
-				done: 'portWizardSubmit',
+				done: self.portWizardComplete,
```

Comment:
Curious why the method is referenced directly when `cancel` references the method's name?

Reply:
I'm trying to update some methods to support references, while keeping the name support for backwards compatibility (i.e. `navigationWizard` instances already in use in CCPro and AppExchange). However I haven't updated the `cancel` method yet. I think I'll take the chance to do it now, because it is the last wizard global method that does not support references.

Reply:
👍  just be sure to take into account the context in which those methods will be called (`this`).